developer-docs-site/docs/tutorials/first-dapp.md

```diff
@@ -11,9 +11,9 @@ For this tutorial, we will use the Move Module `HelloBlockchain` described in [Y
 
 We will use:
 
-- The [Aptos SDK](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/typescript/sdk).
-- The [Aptos Wallet](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/web-wallet), and
-- The [Aptos CLI](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos) to interact with blockchain.
+- The [Aptos Typescript SDK][ts_sdk],
```

Comment:
do we still have links?

Reply:
Yeah this is a funky docusaurus syntax, see the bottom of the file.